### PR TITLE
Validation error and UI consistency update

### PR DESCRIPTION
**Pull request by tonytlwu**
### Changes

- Moves elevation of validation error message
- Disables thumbnail by default until user enables it
- Removes unnecessary DOM elements
- Moves minimal dropdown styles to `fliplet-studio-ui` package ([PR](https://github.com/Fliplet/fliplet-api/pull/2504))

### CODE REVIEW

File: js/interface-directory.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ var DataDirectoryForm = (function() {
     }
 
     if (typeof this.directoryConfig.enable_thumbs === 'undefined' && this.directoryConfig.thumbnail_field !== '') {
-      this.directoryConfig.enable_thumbs = true;
+      this.directoryConfig.enable_thumbs = false;
```
Comment (squallstar) [inline js/interface-directory.js:49]: no thumbnails by default?

File: css/interface.css [hunk 2]
```diff
@@ -11,7 +11,7 @@ body {
   right: 15px;
   left: 15px;
   margin: 0;
-  z-index: 1;
```
Comment (squallstar) [inline css/interface.css:14]: why 3? worth making a comment?

File: js/interface-directory.js (hunk 1 above)
Comment (tonytlwu) [reply js/interface-directory.js:49]: Nope. When enabling thumbnails by default, the interface also asks for a folder. It's best to keep this disabled, and we can enable it for the specific layouts/app templates if necessary.

**Review by squallstar — APPROVED** [review]

File: css/interface.css (hunk 2 above)
Comment (tonytlwu) [reply css/interface.css:14]: The interface contians the `.select-proxy-display` dropdown, for which the `.select-proxy-display .icon` icon element has a `z-index` of 2

Comment (tonytlwu) [reply css/interface.css:14]: This means a `z-index` of 3 is required to ensure the alert message sits on top of everything else

**Pull request merged**